### DISCUSSION TITLE
Questions around PartitionManagerError & PartitionManagerWarning

### DISCUSSION THREAD

**Discussion by mpaul31** [Azure/azure-functions-durable-extension#1834] [Q&A]
I've been looking through our production logs trying to understand some scaling behaviors and I have come across a bunch of the following errors and warning and wanted to see if I can get some more information around this:

- System.ArgumentException: A lease ID must be specified when changing a lease. (Parameter 'accessCondition')
- purchasetaskhub-control-00: Attempted to release control queue purchasetaskhub-control-00, which wasn't being watched!

Also, not sure if it is related, but it seems our app is running into the following HostID collision issue but not 100% sure if this impact durable functions in any way.

https://github.com/Azure/azure-functions-host/issues/2015

Here is some additional info if needed:

East US
auction-etr-int-purchases-east-prod

Like always, thanks!

**Comment by ConnorMcMahon** [comment]
The second of those two is definitely noise that I accidentally introduced in the new partition management strategy. I tried to fix that in my initial attempt to reduce the new costs with this model, but we had to revert that PR at the last moment. Will aim to clean this up soon.

The first one I see intermittently during investigations, but I don't believe it has ever indicated an actual error. I would have to investigate a bit further into why we occasionally don't provide a lease ID.

**Reply by mpaul31** [reply]
OK great thanks for checking on those.

@ConnorMcMahon Have you ran into any customer reports that were caused by the HostID truncation issue I linked above?
